django_sirene/tests/tests_importer.py

```diff
@@ -132,243 +102,400 @@ def test_import_institutions_can_update_db(self):
+        self.assertEqual(institution.activity.code, "AA")
+        self.assertEqual(institution.address, "75  RUE NORBERT DUPONT")
+        self.assertEqual(institution.commercial_name, "INSTITUTION TEST")
+        self.assertEqual(institution.creation_date, date(1970, 1, 1))
+        self.assertEqual(institution.department, "04")
+        self.assertTrue(institution.is_headquarter)
+        self.assertEqual(institution.municipality.code, "4109")
+        self.assertEqual(institution.municipality.name, "NANTES")
+        self.assertEqual(institution.siret, dbo.siret)
+        self.assertEqual(institution.workforce, "5")
+        self.assertEqual(institution.zipcode, "04000")
+        self.assertFalse(institution.is_expired)
+        # will be assigned later
+        self.assertEqual(institution.name, "")
+        self.assertEqual(institution.headquarter, None)
+        self.assertEqual(institution.legal_status, None)
+        # legacy
+        self.assertFalse(institution.is_hidden)
+        self.assertEqual(institution.tel, None)
+
+    def test_remove_update_expire_institution(self):
         dbo = InstitutionFactory()
         row = _get_row_from_object(dbo)
-        row.update({
-            'VMAJ': 'E',
-        })
+        row.update({"etatAdministratifEtablissement": "F"})
         self.assertFalse(dbo.is_expired)
         self.assertFalse(dbo.updated_from_filename)
         old_date = dbo.updated
 
-        CSVImporter([row], filename=self.filename).run()
+        CSVEtablissementImporter([row], filename=self.filename).run()
         dbo.refresh_from_db()
         self.assertTrue(dbo.is_expired)
-        self.assertEqual(dbo.updated_from_filename, self.filename)
         self.assertNotEqual(old_date, dbo.updated)
 
     def test_recreate_an_expired_institution(self):
-        """
-        """
         expired = InstitutionFactory(is_expired=True)
         row = _get_row_from_object(expired)
-        row.update({
-            'VMAJ': 'C',
-        })
+        row.update({"etatAdministratifEtablissement": "A"})
         self.assertEqual(Institution.objects.actives().count(), 0)
-        CSVImporter([row], filename=self.filename).run()
+        CSVEtablissementImporter([row], filename=self.filename).run()
         self.assertEqual(Institution.objects.actives().count(), 1)
 
     def test_update_institution(self):
-        """
-        """
-        chars = '___'
+        chars = "___"
         dbo = InstitutionFactory()
         row = _get_row_from_object(dbo)
-        row.update({
-            'VMAJ': 'F',
-            'NOMEN_LONG': dbo.name + chars
-        })
+        row.update({"enseigne1Etablissement": dbo.name + chars})
         self.assertNotIn(chars, dbo.name)
-        CSVImporter([row], filename=self.filename).run()
+        CSVEtablissementImporter([row], filename=self.filename).run()
         dbo.refresh_from_db()
-        self.assertIn(chars, dbo.name)
+        self.assertIn(chars, dbo.commercial_name)
```

Comment:
Why not `dbo.name`? If this is intended, why not `self.assertNotIn(chars, dbo.commercial_name)` in line 200?

Reply:
That's indeed a mistake ! test_update_institution line 195 has been badly updated, i'm on it